js/interface.js

```diff
@@ -704,25 +566,160 @@ var DynamicLists = (function() {
         }
       });
     },
-    toggleRuleType: function(type, isTypeActive) {
-      var accessRuleIndex = -1;
+    managePermissionList: function(options) {
+      $('.add-entry-checkbox').find('.hidden-settings').toggleClass('active', options.isAddEntryActive);
+      $('.edit-entry-checkbox').find('.hidden-settings').toggleClass('active', options.isEditEntryActive);
+      $('.delete-entry-checkbox').find('.hidden-settings').toggleClass('active', options.isDeleteEntryActive);
+
+      this.toggleRuleType({
```

Comment:
See https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/459#discussion_r539485965

```suggestion
      this.toggleRuleTypes({
```

Reply:
@tonytlwu added suggestion